WooCommerce/Classes/ViewRelated/Orders/Order Filters/FilterOrderListViewModel.swift

```diff
@@ -13,6 +13,7 @@ final class FilterOrderListViewModel: FilterListViewModel {
         let dateRange: OrderDateRangeFilter?
         let product: FilterOrdersByProduct?
         let customer: CustomerFilter?
+        let salesChannel: SalesChannelFilter?
```

Comment:
```suggestion
        let salesChannel: SalesChannelFilter? = nil
```

You don't need to change it now, but for a partial PR like this, it might be better to use the default. It saves you a bunch of changes in the tests, which you're probably going to have to change again when you implement it properly. The compiler will still help you catch them all when you remove the default in a future PR.

Reply:
True 😅 , thanks for pointing it out.